src/Neo.VM/Utility.cs

```diff
@@ -103,10 +70,12 @@ public static long GetBitLength(this BigInteger value)
             if (value.IsZero || value == BigInteger.MinusOne)
                 return 0;
 
-            if (!value.Is256Bit())
+            var ret = (long)Math.Ceiling(BigInteger.Log(value.Sign < 0 ? -value : value + 1, 2.0));
+
+            if (ret > 256)
```

Comment:
This is incorrect. Should watch the video to understand what bit length is. It microsoft conference video call, with the developers who made the `BigInteger.GetBitLength` method.

Reply:
Log works fine. All math libraries use this method. Also its only `+` or `-` 1 if its really huge numbers. `2^(32*int.MaxValue)` that is a number that is 10,000+ characters long.